src/lib/pipeline_configuration.py

```diff
@@ -454,7 +454,92 @@ def clean_age_with_range_filter(text):
                        )
                    ],
                    ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value("kakuma nationality"),
-                   raw_field_fold_strategy=FoldStrategies.assert_equal)
+                   raw_field_fold_strategy=FoldStrategies.assert_equal),
+
+        CodingPlan(raw_field="girls_education_champions_raw",
+                   dataset_name="kakuma_girls_education_champions",
```

Comment:
Does this one need the kakuma prefix?

Reply:
I think this is just used in analysis, and the other analysis-related fields don't mention kakuma

Reply:
Agreed will update in a different PR since this is used in other datasets too.